pkg/genericapiserver/mux/container.go

```diff
@@ -29,8 +29,12 @@ import (
 // handlers that do not show up in swagger or in /
 type APIContainer struct {
 	*restful.Container
+
+	// NonSwaggerRoutes are recorded and are visible at /, but do not show up in Swagger.
 	NonSwaggerRoutes PathRecorderMux
-	SecretRoutes     Mux
+
+	// SecretRoutes are not recorded, are not visible at / and do not show up in Swagger.
+	SecretRoutes *http.ServeMux
```

Comment:
They are all protected in the HandlerContainer.

The SecretRoutes are not recorded because they are secret ;-) They don't show up anywhere.